docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/upgrade-guides/2-0-0-upgrade-guide/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "2.0.0 upgrade guide"
+sidebar_position: -20
+---
+
+## Configuration
+
+BigQuery Loader 2.0.0 brings changes to the loading setup. It is no longer neccessary to configure and deploy three (in [1.X](/docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/previous-versions/bigquery-loader-1.x/index.md) it was loader, repeater and mutator) independent applications in order to load your data to BigQuery.
```

Comment:
Beyond just configuration... we also make obsolete some of the cloud infrastructure.  E.g. the pubsub topic that connected the 3 components.  And the GCS bucket where the repeater loaded failed inserts.

I think we can re-write this paragraph to improve readability.  But please add that extra info first, and then we can improve readability after we've got the content.